### DISCUSSION TITLE
Gitlab Vercel Error: Project not found 404 when trying to import repository

### DISCUSSION THREAD

**Discussion by giancarlosisasi** [vercel/vercel#6839] [Help] [answered]
Hi,
We are trying to deploy a new frontend app using vercel. We have created a vercel team and now we want to deploy an existing GitLab repository. When we try to import the GitLab repo the UI keeps loading/blocked:
(I have hidden the project and team names for privacy reasons)
![bug-vercel-import-project](https://user-images.githubusercontent.com/17274950/137203990-1b42f3c1-5b4c-47cb-b68a-e0783e2968e4.png)

In the devtools I can see 2 requests with 4xx errors:
1. https://vercel.com/api/v1/integrations/integration/oac_xxxxxxxxxxxx/configuration?teamId=xxxxxxx
```
code: "not_found"
message: "Not Found"
```
2. https://vercel.com/api/v2/projects/my-project?teamId=xxxxxxxx
```
code: "not_found"
message: "Project not found"
```

I this a bug in the web or maybe we need to configure the vercel team with a specific option? 🤔 

Thanks in advance.

**Comment by jasondonnette** [comment, marked as answer]
This appears to be happening with github projects too. I think it's an outage of some sort (not showing up on https://www.vercel-status.com/ though)

**Reply by samcx** [reply]
Hi there,

We have recently pushed up a fix for this! We apologize for this inconvenience this has caused.

**Reply by giancarlosisasi** [reply]
@samsisle THANKSSSS
now it works as expected 😄
